Include/cpython/objimpl.h

```diff
@@ -90,9 +90,6 @@ PyAPI_FUNC(int) PyObject_IS_GC(PyObject *obj);
 #  define _PyGC_FINALIZED(o) PyObject_GC_IsFinalized(o)
 #endif
 
-PyAPI_FUNC(PyObject *) _PyObject_GC_Malloc(size_t size);
```

Comment:
``_PyObject_GC_Malloc`` is part of stable ABI. AFAIK the function cannot be removed.

Reply:
It's not part of the stable ABI. It starts with an underscore.
https://www.python.org/dev/peps/pep-0384/#excluded-functions

Reply:
`Misc/stable_abi.txt` define it as stable ABI.

```
function _PyObject_GC_Malloc
    added 3.2
    abi_only
```

On the other hand, no public macro in Python/C API use it. So I doubt it is actually stable abi.

As far as this repo, only one package in top4000 packages uses it.
https://github.com/hpyproject/top4000-pypi-packages/search?q=_PyObject_GC_Malloc